index.html

```diff
@@ -0,0 +1,1345 @@
+                alt="image?url"
+              />
+              <p class="trans-text">75% off</p>
+            </div>
+            <div class="deal-caption-prices">
+              <p class="deal-caption">
+                The All-in-One Adobe Creative Cloud Suite Certification Course
+                Bundle
+              </p>
+              <div class="deal-prices">
+                <p class="old-price">1600</p>
+                <p class="new-price">33.99</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="latest-deals-main">
+            <div class="img-tag-container">
+              <img
+                src="https://cdnp0.stackassets.com/6b0448f38732728d60f8647d2c01a68070c5faf0/store/f5b737d616d2fab13d25f07ded2660409b9f2902a13207743957dd42af48/sale_25931_primary_image.jpg"
+                alt="image?url"
+              />
+              <img
+                class="amarilla"
+                src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/deals-ribbon.svg"
+                alt="image?url"
+              />
+              <p class="trans-text">75% off</p>
+            </div>
+            <div class="deal-caption-prices">
+              <p class="deal-caption">
+                The All-in-One Adobe Creative Cloud Suite Certification Course
+                Bundle
+              </p>
+              <div class="deal-prices">
+                <p class="old-price">1600</p>
+                <p class="new-price">33.99</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer>
+      <div class="lower-inside">
+        <div class="footer-btns">
+          <a href="#">
+            <i class="fa fa-facebook-official" aria-hidden="true"></i>
+          </a>
+          <a href="#"> <i class="fa fa-twitter" aria-hidden="true"></i> </a>
+          <a href="#">
+            <i class="fa fa-youtube-play" aria-hidden="true"></i>
+          </a>
+          <a href="#"> <i class="fa fa-instagram" aria-hidden="true"></i> </a>
+          <a href="#"> <i class="fa fa-pinterest-p" aria-hidden="true"></i> </a>
+          <a href="#"> <i class="fa fa-wifi" aria-hidden="true"></i> </a>
+          <a href="#"> <i class="fa fa-envelope" aria-hidden="true"></i> </a>
+        </div>
+
+        <div class="footer-links">
+          <a href="#"> EVENTS</a>
+          <a href="#"> ABOUT</a>
+          <a href="#"> ADVERTISE</a>
+          <a href="#"> JOBS</a>
+          <a href="#"> CONTACT</a>
+        </div>
+      </div>
+    </footer>
+    <div class="second-footer">
+      <p class="last-text">
+        <img
+          alt="The Next Web"
+          class="footer-legal-logoTNW"
+          src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg"
+        />
+        The Next Web © 2006–2020 The Next Web B.V. Made with love 3 in
+        Amsterdam. Powered by StackPath
+      </p>
```

Comment:
- [ ] `The Next Web The Next Web © 2006–2020 The Next Web B.V. Made with love 3 in Amsterdam. Powered by StackPath `text should in a single line on desktop and tablet screen and it can be 2 or more than 2 lines on the mobile screen.

- [ ] StackPath should be bold.